.github/workflows/CI.yml

```diff
@@ -15,7 +15,7 @@ jobs:
       fail-fast: false
       matrix:
         version:
-          - '1.5'
+          - '1.6'
           - '1'
```

Comment:
As of now `'1'` and `'1.6'` specify the same version number. It is very likely that 1.6 will be the new LTS. If we want to support that in the future as well we should still include it explicitly. Otherwise we could drop it.

Reply:
I had exactly this in mind when keeping `'1.6'` and `'1'`, so I would leave it as is.